spice/training/actions.py

```diff
@@ -118,6 +136,35 @@ def get_device(self):
 
         return device
 
+    def _update_training_round(self, training_round_id, status: str):
+        mutation = gql(
+            """
+            mutation resolveVerifyTrainingRoundFromHardware($trainingRoundId: String!, $status: String!) {
+                resolveVerifyTrainingRoundFromHardware(trainingRoundId: $trainingRoundId, status: $status) {
```

Comment:
if the function is update_training_round the gql mutation should be the same name

```suggestion
    def _update_training_round(self, training_round_id, status: str):
        mutation = gql(
            """
            mutation updateTrainingRoundFromHardware($trainingRoundId: String!, $status: String!) {
                updateTrainingRoundFromHardware(trainingRoundId: $trainingRoundId, status: $status) {
```